test/e2e/cluster.go

```diff
@@ -29,32 +29,30 @@ const (
 var _ = Describe("Cluster", func() {
 	var p project.Project
 
-	var _ = BeforeEach(func() {
+	var _ = BeforeEach(func(ctx context.Context) {
 		By("creating a test namespace")
 		p = project.NewProject(clients.Kubernetes, clients.Project, testNamespace)
-		err := p.Create(context.Background())
+		err := p.Create(ctx)
 		Expect(err).NotTo(HaveOccurred(), "Failed to create test namespace")
 
 		By("verifying the namespace is ready")
-		Eventually(func() error {
-			return p.Verify(context.Background())
-		}).Should(BeNil())
+		Eventually(func(ctx context.Context) error {
+			return p.Verify(ctx)
+		}).WithContext(ctx).Should(BeNil())
 	})
 
-	var _ = AfterEach(func() {
+	var _ = AfterEach(func(ctx context.Context) {
 		By("deleting a test namespace")
-		err := p.Delete(context.Background())
+		err := p.Delete(ctx)
 		Expect(err).NotTo(HaveOccurred(), "Failed to delete test namespace")
 
 		By("verifying the namespace is deleted")
-		Eventually(func() error {
-			return p.VerifyProjectIsDeleted(context.Background())
-		}, 5*time.Minute, 10*time.Second).Should(BeNil())
```

Comment:
Here and in other places polling interval and timeout match default values we set [here](https://github.com/Azure/ARO-RP/blob/6191c11a035462cba7c4bc53b0f9fe18e8868a59/test/e2e/setup.go#L411-L412) so I removed them. Probably makes sense to set only custom values.